tests/lib/snaps/apply-config-hook-set/meta/hooks/apply-config

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+echo "Setting a configuration value"
+if ! snapctl set baz=qux; then
```

Comment:
Can we reuse `snapctl-hooks` instead of adding a new snap?  Doesn't look like it's doing much ATM.